scripts/tests/chiptest/__init__.py

```diff
@@ -150,6 +150,9 @@ def _GetInDevelopmentTests() -> Set[str]:
         "Test_TC_SMOKECO_2_5.yaml",          # chip-repl does not support local timeout (07/20/2023)
         "Test_TC_SMOKECO_2_6.yaml",          # chip-repl does not support local timeout (07/20/2023)
         "Test_TC_IDM_1_2.yaml",              # chip-repl does not support AnyCommands (19/07/2023)
+        "Test_TC_DRLK_2_2.yaml",             # chip-repl does not support EqualityCommands pseudo-cluster (08/04/2023)
+        "Test_TC_DRLK_2_3.yaml",             # chip-repl does not support EqualityCommands pseudo-cluster (08/04/2023)
+        "Test_TC_DRLK_2_12.yaml",            # chip-repl does not support EqualityCommands pseudo-cluster (08/04/2023)
```

Comment:
This is actually disabling this test in CI altogether...  even if you had not removed them from ciTests.json.  Once https://github.com/project-chip/connectedhomeip/pull/28526 merges you should be able to enabled it for chip-tool-python, which is what's wanted here.

Reply:
Tracking issue added: https://github.com/project-chip/connectedhomeip/issues/28545

Reply:
@jrhees-cae Note that https://github.com/project-chip/connectedhomeip/pull/28526 is now merged.  Up to you whether you want to re-enable the tests in this PR or a followup.